README.md

```diff
@@ -5,45 +5,164 @@
 
 <img src="fuse/doc/FuseMedML-logo.png" alt="drawing" width="30%"/>
 
-# What is FuseMedML?
```

Comment:
And about that description, maybe boosting or enabling instead of encouraging.
Maybe we should also mention that it improves collaboration 
Just thoughts that I do not completely agree with :-) so feel free to ignore,

Reply:
done